.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+
+node_js:
+  - "9"
+  - "8"
+  - "6"
+  - "4"
```

Comment:
I would _just_ support 8+ for this, and leverage async_hooks as much as possible. I'm not a fan of https://github.com/othiym23/node-continuation-local-storage. I'm not 100% sure how this is compatible with the Zipkin module. See https://www.npmjs.com/package/cls-hooked.